Source/WTF/wtf/SafeStrerror.cpp

```diff
@@ -36,23 +36,23 @@ namespace WTF {
 CString safeStrerror(int errnum)
 {
     constexpr size_t bufferLength = 1024;
-    char* cstringBuffer = nullptr;
+    std::span<char> cstringBuffer;
     auto result = CString::newUninitialized(bufferLength, cstringBuffer);
 #if OS(WINDOWS)
-    strerror_s(cstringBuffer, bufferLength, errnum);
+    strerror_s(cstringBuffer.data(), cstringBuffer.size(), errnum);
 #else
-    auto ret = strerror_r(errnum, cstringBuffer, bufferLength);
+    auto ret = strerror_r(errnum, cstringBuffer.data(), cstringBuffer.size());
     if constexpr (std::is_same<decltype(ret), char*>::value) {
         // We have GNU strerror_r(), which returns char*. This may or may not be a pointer into
         // cstringBuffer. We also have to be careful because this has to compile even if ret is
         // an int, hence the reinterpret_casts.
         char* message = reinterpret_cast<char*>(ret);
```

Comment:
Just noticed the comment above about this having to compile if `ret` is `int` in passing and it seems wrong.